guide/ops/locations/_clouds.md

```diff
@@ -278,7 +278,10 @@ The name that Apache Brooklyn generates for your virtual machine will, by defaul
 If you have created a lot of virtual machines, this kind of naming may not be helpful. This can be changed using the following YAML in your location's `brooklyn.config`:
```

Comment:
I think we should change the wording in this doc. I think it's wrong to say "this kind of naming may not be helpful". The point is more that:

    However, some organization have their own naming convention for VMs. The naming strategy used by
    Brooklyn can be configured using the following YAML in your location's `brooklyn.config`:

If you have created a lot of virtual machines, this kind of naming may not be helpful. This can be changed using the following YAML in your location's brooklyn.config: